### PR TITLE
Fix the scripts for macos

### LINKED ISSUE #268: Can't start the test server on mac
The docker tries to bind to /var/ which throws an error on mac. This isn't mission critical but would be nice to fix going forward

### PR DESCRIPTION
**Pull request by JackWilb**
Closes #268

### CODE REVIEW

File: client/scripts/start.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ realpath() {
 FILE_PATH=$(dirname $(realpath "$0"))
 source $FILE_PATH/../../.env.test
 
-nohup yarn serve --port 58080 >server.out &
+FLASK_SERVE_PORT=$FLASK_SERVE_PORT nohup yarn serve --port 58080 >server.out &
 echo $! >server.pid
```
Comment (AlmightyYakob) [inline client/scripts/start.sh]: Why is it required to do this? Since we're already sourcing `.env.test` isn't this essentially a no op?

Comment (JackWilb) [reply client/scripts/start.sh]: It shouldn't be, I'm just troubleshooting. Having some issues with sourcing the .env.test on the ubuntu runner. I'm spinning up a local ubuntu instance here to test

**Comment by waxlamp** [comment]
With the new style of local testing, I think this can be closed without further action. @JackWilb, if not, please reopen the issue with some discussion.

**Comment by JackWilb** [comment]
These issues remain. Starting the test server causes issues with the readlink command and the mounting paths for the arango data. I think it might be better to start fresh from a new branch and just fix those 2 issues though since this was quite a large change for a small problem

**Comment by waxlamp** [comment]
> These issues remain. Starting the test server causes issues with the readlink command and the mounting paths for the arango data. I think it might be better to start fresh from a new branch and just fix those 2 issues though since this was quite a large change for a small problem. Let me know what you think, @waxlamp

Ok. Can you update the description of #268 with a screencap or transcript of the exact commands you're running and the exact error messages that result? You shouldn't have any problems with `readlink` if you're following the README instructions, so it's important to know exactly what is leading to your error conditions.

**Pull request closed without merging**